gtd/config.js

```diff
@@ -0,0 +1,21 @@
+const {
+  constants: { TIMEFRAMES },
+} = require('@notify-watcher/core');
+
+module.exports = {
+  name: 'gtd',
+  displayName: 'GTD',
+  description: 'GTD Manquehue plans watcher',
+  auth: false,
+  libs: ['axios', 'cheerio', 'lodash'],
+  timeframe: {
+    type: TIMEFRAMES.day,
+    hour: 10,
+  },
+  notificationTypes: {
+    newPlan: {
+      key: 'newPlan',
```

Comment:
Should these have display names as well?

Reply:
(A description could probably be a long paragraph...)

Reply:
yeah they should, will leave an issue for that

Reply:
_some_ should, others don't need it

Reply:
opened #26 for this